rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -64,7 +64,7 @@
       "form": {
         "validation": {
           "messages": {
-            "pattern": "Should be in the ISO 8601, YYYY-MM-DD."
+            "patternMessage": "Should be in the ISO 8601, YYYY-MM-DD."
```

Comment:
Maybe the same message as in https://github.com/rero/rero-ils/pull/1021/files#diff-012a8082b0a21ed9db84b8553aac2c6eR80 ?